astropy/wcs/setup_package.py

```diff
@@ -296,47 +296,5 @@ def get_extensions():
     return [Extension('astropy.wcs._wcs', **cfg)]
 
 
-def get_package_data():
-    # Installs the testing data files
-    api_files = [
-        'astropy_wcs.h',
-        'astropy_wcs_api.h',
-        'distortion.h',
-        'isnan.h',
-        'pipeline.h',
-        'pyutil.h',
-        'sip.h',
-        'util.h',
-        'wcsconfig.h',
-        ]
-    api_files = [join('include', 'astropy_wcs', x) for x in api_files]
-    api_files.append(join('include', 'astropy_wcs_api.h'))
-
-    wcslib_headers = [
-        'cel.h',
-        'lin.h',
-        'prj.h',
-        'spc.h',
-        'spx.h',
-        'tab.h',
-        'wcs.h',
-        'wcserr.h',
-        'wcsmath.h',
-        'wcsprintf.h',
-        ]
-    if not setup_helpers.use_system_library('wcslib'):
-        for header in wcslib_headers:
-            source = join('cextern', 'wcslib', 'C', header)
-            dest = join('astropy', 'wcs', 'include', 'wcslib', header)
-            if newer_group([source], dest, 'newer'):
-                shutil.copy(source, dest)
```

Comment:
I need to figure out how to reproduce this behavior with ``setup.cfg``, this is what's causing the test failure currently

Reply:
(and as a side note, the reason I didn't pick this up sooner is that the test that exercises this is set to skip if certain env variables aren't set, and this was the case with tox since env vars have to be whitelisted)

Reply:
Hmm, this looks all rather complicated; is the library created in `astropy.wcs` used outside of astropy?

Reply:
The idea was to make it so that other packages could create C extensions linked against the bundled WCSLIB in astropy but not sure if anyone uses it!

Reply:
I don't think we document this?

Reply:
Ah we do somewhat: http://docs.astropy.org/en/stable/api/astropy.wcs.get_include.html#astropy.wcs.get_include

Reply:
I see, so it is a bit like `numpy.get_include()`. Though indeed we should then make sure it always points to the correct include files, either that of the system or of astropy.